### ISSUE TITLE
Request: Can you implement Drone Remote Operating System/Smart Robotic Control System on MegaMek?

### ISSUE THREAD

**Issue by binaryspica** [MegaMek/megamek#1366]
### Environment
Ver 0.45.4
Windows 7

### Description
For now, although MegaMekLab has Drone Remote Operating System and Smart Robotic Control System, but it doesn't implemented on MegaMek. As always, living pilot(s) are pilot the unit, whatever the unit has the device or not. For Drones, the presence of a friendly unit with Drone Carrier Control System does nothing either. I wonder it is implemented or just a mistake of mine and have tried it, but the conclusion is it is not implemented.

So, can you make the option to these devices functional someday? I don't think that it is that easy, but I hope that it works someday.... 

And, it seems that make the robotic control seems difficult, for robotic systems needs complicated stances....

For reference:

Drone (Remote) Carrier Control System/Operating System: pg 305~306, TO
Robotic rules: pg 134~159, IO

**Comment by mkerensky** [comment]
We intend to implement all the equipment and all the rules in the most current set of rulebooks.  "Someday" is a pretty accurate time estimate, but this is on the plan with all the other unimplemented rules.

**Comment by binaryspica** [comment]
Yeah, it seems complicated so I don't think that it would be released soon.

**Comment by snowarc** [comment]
Id love this too really!
The same thing for the aerospace fighters too
